src/daemon/abrt-action-save-package-data.c

```diff
@@ -449,6 +451,8 @@ static int SavePackageDescriptionToDebugDump(const char *dump_dir_name, const ch
 
     if (component)
         dd_save_text(dd, FILENAME_COMPONENT, component);
+    if (sub_components)
+        dd_save_text(dd, FILENAME_SUBCOMPONENTS, sub_components);
```

Comment:
Again, what is the point? The only time you need to know the subcomponents are when sending the report. I.e. you do just-in-time negotiation.